packages/create-docusaurus/templates/classic/docusaurus.config.js

```diff
@@ -1,10 +1,23 @@
 // @ts-check
-// Note: type annotations allow type checking and IDEs autocompletion
+// Note: '@type' annotations throughout allow type checking and IDEs autocompletion
+
+// NOTE:  In this default config, we declare a JS constant 'const config ='
+// to further help your IDE give TypeScript hints while editing this file.
+// 'module.exports =' at the very bottom is the syntax that uses
+// that 'config' constant.  There is not a single way/syntax to declare
+// your config within this file.  In fact, you have the full power of
+// Node.js, as long as you return an object that matches the shape we
+// expect with 'module.exports =' (or a function that creates that object).
+// Our docs give examples of putting things inside 'module.exports = { }'
+// which in this file is equivalent to putting into 'config = { }' instead,
+// It's completely your choice.
+// See also: https://docusaurus.io/docs/api/docusaurus-config#overview
```

Comment:
I don't really like the idea to include a wall of text like that for all new users, most of them are familiar with nodejs and it's useless in many cases

I'm more in favor of adding very simple but convenient link to https://docusaurus.io/docs/api/docusaurus-config#overview, and improving a bit the docs of this section. But again it doesn't have to be that verbose. You can just link to another page that explains how CJS modules work in Node.js and show 2 equivalent examples with/without the constant.

Reply:
what about just lines 11-14?  It would have helped me and that's the essence... the mismatch of what examples show, versus what folks see in this default docusaurus.config.js  So, those 11-13 lines satisfy me, and line 14 satisfy you.  In favor of removing lines 4-10.  Deal?

Reply:
```jsx
// There are various equivalent ways to declare your Docusaurus config.
// https://docusaurus.io/docs/api/docusaurus-config
```

That seems largely good enough for me. Confused users can just follow the link and have more explanations there